static/templyte/src/app/tutorial/tutorial.component.html

```diff
@@ -0,0 +1,82 @@
+<mat-toolbar class="mat-elevation-z3">
+    <nav class="navbar">
+        <ul class="nav navbar-right">
+            <li>
+                <button mat-button (click)="goHome()">Home</button>
+            </li>
+            <li>
+                <button mat-button (click)="createTemplate()">Create Template</button>
+            </li>
+            <li>
+                 <button mat-button disabled>Tutorial</button>
+            </li>
+            <li>
+                <button mat-button (click)="myTemplates()">My Templates</button>
+            </li>
+            <li>
+                <button mat-button (click)="logout()">Logout</button>
+            </li>
+        </ul>
+    </nav>
+</mat-toolbar>
+
+<html>
+<head>
+
+<title>Page Title</title>
+</head>
+<body>
+<h1>Create a Template</h1>
+
+<ol type="1">
+  <li>Navigate to the "Create Template" page from the navigation bar at the top of each page</li>
+  <img src="assets/img/tutorial/menu.jpg" alt="Menu" width="533" height="44" >
```

Comment:
You should not have image elements as children of an unordered list - this violates html standards

Reply:
This is an [ordered list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol), can you please provide me with documentation of the html standards to show that this is actually against them. The images are also not children of the list.

Reply:
Permitted content: Zero or more <li> elements, which in turn often contain nested <ol> or <ul> elements.'